certbot/plugins/selection.py

```diff
@@ -160,6 +161,9 @@ def choose_configurator_plugins(config, plugins, verb):
 
     req_auth, req_inst = cli_plugin_requests(config)
 
+    installer_question = ("How would you like to authenticate and install "
```

Comment:
Is it correct to mention authentication here?

Reply:
Oh, right. It's not :)